pokemongo_bot/cell_workers/sniper.py

```diff
@@ -0,0 +1,370 @@
+from __future__ import unicode_literals
+
+import os
+import time
+import json
+import requests
+
+from random import uniform
+from pokemongo_bot import inventory
+from pokemongo_bot.inventory import Pokemon, Pokemons
+from pokemongo_bot.base_task import BaseTask
+from pokemongo_bot.base_dir import _base_dir
+from pokemongo_bot.constants import Constants
+from pokemongo_bot.worker_result import WorkerResult
+from pokemongo_bot.cell_workers.utils import format_dist, format_time, fort_details
+from pokemongo_bot.walkers.walker_factory import walker_factory
+from pokemongo_bot.cell_workers.pokemon_catch_worker import PokemonCatchWorker
+
+POKEBALL_ID = 1
+GREATBALL_ID = 2
+ULTRABALL_ID = 3
+
+class OrderMode():
+    IV = 'iv'
+    VIP = 'vip'
+    MISSING = 'missing'
+    THRESHOLD = 'threshold'
+    VALUES = [IV, VIP, MISSING, THRESHOLD]
+    DEFAULT = [MISSING, VIP, THRESHOLD]
+
+class SnipingMode():
+    URL = 'url'
+    SOCIAL = 'social'
+    VALUES = [URL, SOCIAL]
+    DEFAULT = SOCIAL
+
+class ResponseMapper():
```

Comment:
Why is it strange? From what I've heard by other python fans, it's a good practice. Python is all based on dictionaries, so keeping a constant key name should make things less confusing for maintainances.